src/components/EditorSideBar/EditorSideBarContent.tsx

```diff
@@ -32,22 +41,25 @@ const SideBarContainer = (props: Props) => {
     },
   });
 
+  const { data: rawTemplateData } = useGetTemplatesQuery("");
+
   return (
     <Box
       background="background.secondaryLight"
-      width="280px"
+      padding="24px"
       height="calc(100vh - 72px)"
```

Comment:
Remove the width because the template list page has wider sidebar content to display, the original width is designed only for folders and blueprints which have the same width. The padding problem can be solved when we have empty page design for all sidebar items.